src/vmm/src/lib.rs

```diff
@@ -584,11 +602,11 @@ impl Vmm {
             DEFAULT_ADDRESSS_ALIGNEMNT,
             DEFAULT_ALLOC_POLICY,
         )?;
-
+        let irq = self.irq_allocator.next_irq()?;
         let mmio_range = mmio_from_range(&range);
         let mmio_cfg = MmioConfig {
             range: mmio_range,
-            gsi: 5,
+            gsi: irq,
```

Comment:
Can we check in some test (or add a new one) that the irq values are indeed the ones that were hardcoded before (e.g. check that after adding a block device, its irq is 5).